src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -239,6 +240,10 @@ class ConfigureTaskRunnerAction extends Action {
 					contentPromise = TPromise.as(selection.content);
 				}
 				return contentPromise.then(content => {
+					let editorConfig = this.configurationService.getConfiguration<any>();
+					if (editorConfig.editor.insertSpaces) {
+						content = content.replace(/(\n)(\t+)/g, (_, s1, s2) => s1 + strings.repeat(' ', s2.length * editorConfig.editor.tabSize));
```

Comment:
I keep "\n", in `(_, s1, s2) => s1 + strings.repeat(' ', s2.length * editorConfig.editor.tabSize)` I get `_` as a `$0` match, `s1` as `$1=\n` match and `s2` as `$2=\t+`.

I use `\n` in commentaries just to be sure that I am not going to break any weird cases, where tab character is needed somewhere in command or somewhere in the middle.